src/db/migrations/20230618104332_feature_flags.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ export function up (knex) {
+  return knex.schema
+    .createTable('feature_flags', table => {
+      table.string('name').primary().unique()
```

Comment:
Does this knex magic auto-index this column?
I believe this was the only column that we're querying w/ `where=` clauses.

Reply:
Yes, a primary key index should be created